kolibri/plugins/learn/assets/src/modules/examViewer/handlers.js

```diff
@@ -55,61 +55,63 @@ export function showExam(store, params, alreadyOnQuiz) {
             contentNodes => {
               if (shouldResolve()) {
                 // If necessary, convert the question source info
-                const question_sources = convertExamQuestionSourcesToV3(exam, { contentNodes });
```

Comment:
Not here, but in the unedited code above, couldn't this use the updated util function that you made in this PR that fetches the exam and the content nodes all at once?